pkg/kubevirt/apis/provider_spec.go

```diff
@@ -48,10 +48,10 @@ type KubeVirtProviderSpec struct {
 	// +optional
 	Networks []NetworkSpec `json:"networks,omitempty"`
 	// Region is the name of the region for the VM.
-	Region string
-	// Zones is a list of availability zones for the VM.
+	Region string `json:"region"`
+	// Zone is the name of the zone for the VM.
 	// +optional
-	Zones []string
+	Zone string `json:"zone,omitempty"`
```

Comment:
@stoyanr zone should not be optional in that case and we should add validation if the value is set (the same for region), however, I suggest to merge it like it is and I will fix it in my PR since I refactored validator.

Reply:
Yes, you are right, thanks!